src/unix/signal.c

```diff
@@ -563,8 +562,9 @@ static void uv__signal_stop(uv_signal_t* handle) {
     rem_oneshot = handle->flags & UV_SIGNAL_ONE_SHOT;
     first_oneshot = first_handle->flags & UV_SIGNAL_ONE_SHOT;
     if (first_oneshot && !rem_oneshot) {
-      ret = uv__signal_register_handler(handle->signum, 1);
+      int ret = uv__signal_register_handler(handle->signum, 1);
```

Comment:
Can you undo this? Libuv generally splits assignment from definition. Just the cast to void should be sufficient.